pkg/resources/apiserver/deployment.go

```diff
@@ -316,6 +316,7 @@ func getApiserverFlags(data *resources.TemplateData, etcdEndpoints []string, ena
 		"--requestheader-extra-headers-prefix", "X-Remote-Extra-",
 		"--requestheader-group-headers", "X-Remote-Group",
 		"--requestheader-username-headers", "X-Remote-User",
+		"--endpoint-reconciler-type", "none",
```

Comment:
The problem with this change is the same as we already discussed for addons. Let's say users disable auto-upgrades, so they can better plan upgrades. The existing 1.20 clusters will be reconciled with this flag, which might break those clusters. If we want to give some more time just to be sure, we should probably revert this change.

Reply:
> users disable auto-upgrades

With users you mean KKP admins, right? Or can endusers simply reject our auto update rules?